webpack/components/Table/EmptyStateMessage.js

```diff
@@ -44,6 +46,22 @@ const EmptyStateMessage = ({
       emptyStateBody = error?.response?.data?.displayMessage || __('Something went wrong! Please check server logs!');
     }
   }
+
+  const secondaryActionText = searchIsActive ? __('Clear search') : __('Clear filters');
+
+  const handleClick = () => {
+    if (searchIsActive && !filtersAreActive) {
+      updateSearchQuery('');
+    }
+    if (!!filtersAreActive && !searchIsActive) {
+      resetFilters();
+    }
+    if (searchIsActive && !!filtersAreActive) {
+      resetFilters();
+      updateSearchQuery('');
+    }
```

Comment:
I think this can still be simplified more.

If `searchIsActive` you always call `updateSearchQuery('');`
if `filtersAreActive` you always call `resetFilters()`

So could we rewrite it without using any `&&`?

Reply:
I mean the "call-to-action" empty state completed in the last PR. That flashes on the screen.

Reply:
Would you mind hopping on to the quick call after our standup since I still did not get what you mean? On my end, it seems that nothing is flashing.